receiver/udplogreceiver/udp_test.go

```diff
@@ -22,6 +22,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/operator/input/udp"
```

Comment:
Hm.. this must be part of the third import group. I'm curios why linter wan't triggered

Reply:
Yeah, strange. Looking into it.

Reply:
I opened #13318 to track